.tugboat/config.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  backstage:
+    image: tugboatqa/node:lts
+    commands:
+      init:
+        - yarn
+        - yarn build
```

Comment:
is that needed? hm i thought it would be like yarn tsc here but i'm frankly not sure

Reply:
Would be needed to serve the frontend, although there's no config set up for that to work really ^_>. `yarn tsc` does indeed need to be called first as well.